src/app/root.js

```diff
@@ -14,6 +14,9 @@ function rootReducer(state = {}, action) {
       case actionsList.storeVisitURL:
         draftState.visitURL = action.payload;
         break;
+      case actionsList.LogOut:
+        draftState.LogOut = action.payload;
```

Comment:
We don't actually want to store any payload but instead want to clear the stored user data.

You can see here https://github.com/canonical-web-and-design/jaas-dashboard/blob/master/src/app/selectors.js#L245 that the `isLoggedIn` checks for the existence of the controller and bakery. So you'll want to invalidate this check. I'd recommend nulling out the `bakery` instance. This will cause the application to re-render and when the login check is made it'll return false, logging the user out.

The application doesn't actually support having the bakery ripped out from under it so both this reducer and how the application creates the bakery instances will need to be updated afterwards as a follow-up.